Makefile

```diff
@@ -9,19 +9,23 @@ PYTHON=$(VENV_NAME)/bin/python3
 PACKAGE_NAME=co.nimblehq.flutter.template
 PROJECT_NAME=flutter_templates
 APP_NAME=Flutter Templates
+APP_VERSION=0.4.0
```

Comment:
Just curious, why do we use `0.4.0` instead of `0.1.0` and the `BUILD_NUMBER` is not `1` 🙏

Reply:
@doannimble according to [this comment](https://github.com/nimblehq/flutter_templates/pull/31#discussion_r802533632), I will use an example app version `0.1.0` and build number `1` for this. Will update 🙏